jakarta.ls/src/main/java/org/eclipse/lsp4jakarta/commons/SnippetRegistry.java

```diff
@@ -367,4 +406,24 @@ protected static String findExprBeforeAt(String text, int offset) {
         }
         return expr.toString();
     }
+
+    private static void getMatchedVariables(String line, int start, String[] vars,
```

Comment:
Suggestion for comment here, it wasn't immediately clear to me what this method does.

Reply:
hmm..., usually no comments for private method, the API is finding the matched variables from snippet text, bad method name or you want comments on implementation?

Reply:
I was also confused about the int start variable and the subsequent recall so a few sentences would be nice.

Reply:
If you can understand `replace` API call, It's a pretty similar idea. `start` is the start position of snippet text that the search from.

Reply:
Anyway, as you guys preferred, I added a comment to the method